test/ctia/test_helpers/core.clj

```diff
@@ -47,44 +51,56 @@
   (binding [*config-transformers* (conj *config-transformers* tf)]
     (body-fn)))
 
-(def with-properties-vec
-  (mth/build-with-properties-vec-fn PropertiesSchema))
+(s/defn with-properties*
+  [properties-vec :- (s/pred vector?)
```

Comment:
`vector?` could be refined to assert the properties in the docstring above (`even-sized [...]`)

Reply:
Yes, evenness is asserted a few lines down.